include/internal/attester.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2021 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _INTERNAL_ATTESTER_H
+#define _INTERNAL_ATTESTER_H
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <librats/err.h>
+#include <librats/core.h>
+
+#define RATS_ATTESTERS_DIR "/usr/local/lib/librats/attesters/"
```

Comment:
The install lib path is `set(RATS_INSTALL_LIB_PATH "${RATS_INSTALL_PATH}/lib/rats")`.

`RATS_ATTESTERS_DI`R should consistent with `RATS_INSTALL_LIB_PATH/attesters/`